### PR TITLE
Remove unnesecary code.

### PR DESCRIPTION
**Pull request by DitherWither**
Lets be honest, none of this code is contributing to society in any means whatsoever. Replaced with more useful code.

To run this, first run `make forkbomb` and then run `./forkbomb`

### CODE REVIEW

**Comment by gagan-lohia** [comment]
This is over-engineering  @DitherWither

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,3 @@
-.DS_Store
-
+*
+!README.md
+!forkbomb.c
```
Comment (arledesma) [inline .gitignore:3]: Please see [3.206 Line](https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap03.html#tag_03_206)

> ## 3.206 Line  
> A sequence of zero or more non- <newline> characters plus a terminating <newline> character.

```suggestion
!forkbomb.c
```

**Review by arledesma — COMMENTED** [review]
We have standards

**Comment by tokozedg** [comment]
When this is get merged? I can reproduce.

**Comment by SamZhang02** [comment]
lgtm

**Pull request closed without merging**